src/canopy_server/app.py

```diff
@@ -83,7 +85,7 @@
 
 
 @app.post(
-    "/context/chat/completions",
```

Comment:
Again, you need to change that in `cli.py` as well (but wait for #170 to merge first!)

Reply:
Think im' done, resolving this for the future review